### DISCUSSION TITLE
Getting error in mysql connection string

### DISCUSSION THREAD

**Discussion by RPooja08** [signalfx/splunk-otel-collector#2874] [Q&A] [answered]
We have configured the mysql db monitoring via splunk-otel-collector on a linux machine. but we are getting error in logs.
Please suggest on this.

dial tcp xxx.xx.xx.xx:3306: connect: connection refused

**Comment by atoulme** [comment, marked as answer]
This error indicates that the collector cannot connect to mysql because the connection is refused. The username and password used might be incorrect. Please see this StackOverflow thread for possible solutions: https://stackoverflow.com/questions/13032270/mysql-connection-refused-when-trying-to-connect-to-localhost-using-remote-ip
